spanner/cloud-client/src/test/java/com/example/spanner/LoadCsvExampleIT.java

```diff
@@ -0,0 +1,145 @@
+/*
```

Comment:
Could you add some tests with the CSV files here: https://github.com/cloudspannerecosystem/sampledb?
Also look at the Dataflow job to see if it has any sample CSV.

Reply:
Added tests for a subset of the data in those CSV files since the original files exceed 20000 mutations